### PR TITLE
Duck-typing

### PR DESCRIPTION
**Pull request by piranna**
This pull-request use duck-typing to check is an object is a `File`-like object since ReactNative don't have the concept of `File` or `Blob` objects (`Blob` objects will be supported on ReactNative 0.54, but not for older versions).

It also restore the commits history, do some minor clean-ups on docs and `package.json`, and update the project dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # object-to-formdata
-  A convenient JavaScript function that converts an object to a FormData instance.
+A convenient JavaScript function that converts an object to a FormData instance
 
-```
+```js
```
Comment (therealparmesh) [inline README.md]: 💯

**Comment by piranna** [comment]
You are welcome :-)

**Comment by therealparmesh** [comment]
Manually merged this is. Marking this as closed.

**Comment by piranna** [comment]
Thank you :-)

**Pull request closed without merging**